maven/pom-core.xml

```diff
@@ -12,6 +12,32 @@
   <description>Thin Java abstractions for the standard built-in objects for Javascript.</description>
   <url>https://www.gwtproject.org</url>
 
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <scm>
+    <connection>scm:git:https://github.com/google/elemental2.git</connection>
+    <developerConnection>scm:git:git@github.com:google/elemental2.git</developerConnection>
+    <url>https://github.com/google/elemental2</url>
+  </scm>
+
+  <issueManagement>
+    <url>https://github.com/google/elemental2/issues</url>
+    <system>GitHub Issues</system>
+  </issueManagement>
+
+  <developers>
+    <developer>
+      <id>elemental2</id>
+      <name>The Elemental2 Developers</name>
+    </developer>
+  </developers>
```

Comment:
Do you really need this "developers" section? It doesn't bring any useful information

Reply:
Yep - to upload to Maven Central you need a developers section with at least one developer in it

Reply:
GWT (and others projects like auto-value) doesn't have that information and are able to deploy on Maven central.

Anyway, if we add this section use something like 
```xml
 <developers>
    <developer>
      <name>j2cl team</name>
      <organization>Google</organization>
      <organizationUrl>http://www.google.com</organizationUrl>
    </developer>
  </developers>
```

Reply:
I vaguely recall the developers section not being required and some of my older projects certainly did not include it. It is unclear to me when this started being enforced or how it is enforced. There is some new artifacts published that seem to be able to get away with not including the section - although those I have spotted either come from a "blessed repository" that is automatically added to Maven Central or inherit from the `oss-parent`. This may have something to do with it or it may be that they just started enforcing this.

Certainly if I try and publish anything into my own namespace via https://oss.sonatype.org the process will add the requirement.

Reply:
developers section updated...